docs/en/deployment.md

```diff
@@ -55,6 +60,39 @@ This automated AWS CloudFormation template deploys the solution in the AWS Cloud
 
 You can view the status of the stack in the AWS CloudFormation Console in the **Status** column. You should receive a CREATE_COMPLETE status in approximately 10 minutes.
 
+### Deploy solution from source code
```

Comment:
still prefer to add dev steps as part of source code, it's useless for doc reader

Reply:
changed